declarations_site/catalog/elastic_models.py

```diff
@@ -99,80 +94,79 @@ def get_family_members(self):
                     yield member["family_name"]
 
 
-
-class Declaration(AbstractDeclaration):
+class Declaration(DocType, RelatedDeclarationsMixin):
```

Comment:
Aha, I see that it's populated by a separate command and not with the import commands, which isn't really good as you have to remember to rerun this command on every imported batch. Also I don't see it being used for anything specific. "Deep" search utilises `_all` instead. Was it supposed to use `ft_src`? Shall I change that too?